yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -573,19 +573,19 @@ export class Sequencer {
       );
 
       if (this.isFlushing) {
-        this.log.verbose(`Flushing completed`);
+        this.log.info(`Flushing completed`);
       }
 
       const txHashes = validTxs.map(tx => tx.getTxHash());
 
       this.isFlushing = false;
-      this.log.verbose('Collecting attestations');
+      this.log.info('Collecting attestations');
```

Comment:
@ludamad I'm actually not seeing these logs under in loki even with debug on. Thoughts? I'm not sure if it is all `verbose` logs, or what. That was part of the impetus for upping the level.

Reply:
I filed https://github.com/AztecProtocol/aztec-packages/issues/9958

Reply:
Unfortunately I am not sure. We should write good tests along with moving to a non-node debugger (winston? pino?)